src/mono/Directory.Build.props

```diff
@@ -1,12 +1,69 @@
 <Project>
-
   <PropertyGroup>
     <SkipImportArcadeSdkFromRoot>true</SkipImportArcadeSdkFromRoot>
   </PropertyGroup>
   <Import Project="..\..\Directory.Build.props" />
+  
+  <!-- Set default Configuration and Platform -->
+  <PropertyGroup>
+    <OSGroup Condition="'$(OSGroup)' == '' and '$([MSBuild]::IsOSPlatform(Windows))' == 'true'">Windows_NT</OSGroup>
```

Comment:
Can this instead be shared by libraries, coreclr and mono?

Reply:
not until we have merged @Anipik's configuration system PR.